sklearn/utils/_plotting.py

```diff
@@ -235,80 +231,60 @@ def _get_line_kwargs(
                     )
                 )
 
-        line_kwargs = []
-        for fold_idx, label in enumerate(labels):
-            default_line_kwargs["label"] = label
-            line_kwargs.append(
-                _validate_style_kwargs(default_line_kwargs, fold_line_kwargs[fold_idx])
-            )
-        return line_kwargs
+        if roc_line_kwargs is None and n_curves > 1:
+            default_roc_line_kwargs = {"alpha": 0.5, "linestyle": "-", "color": "blue"}
+        else:
+            default_roc_line_kwargs = {}
 
-    # TODO: if useful for non binary displays (e.g.,`LearningCurveDisplay`,
-    # `ValidationCurveDisplay`) amend to function
-    @classmethod
-    def _validate_line_kwargs(
-        cls,
-        n_curves,
-        fold_line_kwargs=None,
-        default_line_kwargs=None,
-        aggregate_error=False,
-    ):
-        """Ensure `fold_line_kwargs` length and incorporate default kwargs.
-
-        * If `fold_line_kwargs` is None:
-          * If `default_line_kwargs` is None, list of `n_curves` empty dictionaries
-            is returned.
-          * If `default_line_kwargs` is not None, list of `n_curves` dictionaries
-            of `default_line_kwargs` returned.
-        * If `fold_line_kwargs` is a single dictionary, it is incorporated with
-          `default_line_kwargs` using `_validate_style_kwargs`, and the resulting
-          dictionary is repeated `n_curves` times and returned.
-        * If `fold_line_kwargs` is a list of length `n_curves`, each dict is
-          incorporated with `default_line_kwargs` using `_validate_style_kwargs` and
-          returned as list of `n_curves` dictionaries.
-
-        If `fold_line_kwargs` is a list not of length `n_curves`, an error is raised.
+        default_roc_line_kwargs = [
+            {**default_roc_line_kwargs, "label": label} for label in labels
+        ]
```

Comment:
Answering this comment https://github.com/scikit-learn/scikit-learn/pull/30399#issuecomment-2788018265

> I don't see the actual default parameters being passed in (your) PR..?

default curve kwargs are incorporated there, in `get_line_kwargs`

Reply:
Ah yes I see it. I was trying to avoid 'hard coding' the default kwargs in `plot`, for the case where the user is using `plot` independent of `from_cv_results`, in which case these default kwargs do not necessarily always make sense.

They can of course pass their own kwargs in, in this case, so I am okay with this solution, if we are trying to avoid double validation...